### PR TITLE
add list = list.unique() ; aars = aars.unique()

### PR DESCRIPTION
**Pull request by qiujiayu**
add list = list.unique() ; aars = aars.unique(), to remove the duplicate value.

### CODE REVIEW

**Comment by ksoichiro** [comment]
Thank you for your PR.
It seems some tests are failing, so something should be changed.
I'll look into it later.

Removing duplicate values sounds good,
but I don't understand the actual issue..
Could you explain?
I'd like to add some tests for it.

File: plugin/src/main/groovy/com/github/ksoichiro/eclipse/aar/GenerateTask.groovy [hunk 1]
```diff
@@ -392,6 +392,7 @@ target=${extension.androidTarget}
 
         List<String> list = projectDependencies[p]?.collect { it.getQualifiedName().replaceFirst('^:', '') }
         list = list?.findAll { projectNames.find { prj -> prj == it } == null }
+        list = list.unique()
```
Comment (ksoichiro) [inline plugin/src/main/groovy/com/github/ksoichiro/eclipse/aar/GenerateTask.groovy]: I think the `list` sometimes becomes `null` and it causes `NullPointerException` on tests.
Could you change this to `list?.unique()`?

Comment (qiujiayu) [inline plugin/src/main/groovy/com/github/ksoichiro/eclipse/aar/GenerateTask.groovy]: I just want to remove duplicate values;

**Comment by qiujiayu** [comment]
I have change list = list.unique() to list?.unique()

**Comment by ksoichiro** [comment]
OK, thank you!

**Pull request merged**